src/sonic-yang-models/yang-models/sonic-system-aaa.yang

```diff
@@ -30,8 +30,12 @@ module sonic-system-aaa {
                 }
 
                 leaf login {
-                    type string;
-                    description "AAA authentication/authorization/accounting methods - local/tacacs+/disable";
+                    type string {
+                        pattern 'radius|tacacs\+|local|default' {
```

Comment:
This pattern need update to support multiple login method, for example following method will try authentication with tacacs+ protocol first, if tatacs+ authentication failed, will try use local authentication:
tacacs+,local

Reply:
Suggest use this regedx for pattern: ((tacacs\+|local|redus|default)\,{0,1})+

Reply:
Fixed.